autonomy/deploy/generators/docker_compose/templates.py

```diff
@@ -90,6 +90,7 @@
     container_name: abci{node_id}
     image: {runtime_image}
     environment:
+      - PYTHONHASHSEED=0
```

Comment:
We need to disable hash randomization in order to have reproducible ML results among the agents.